e_commerce/settings.py

```diff
@@ -155,6 +151,61 @@
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+
+    # formatting the logs
+    'formatters': {
+        'standard': {  # fomatter name
+            'format': "{asctime} {levelname} {module} {message}",
+            'style': '{'  # this should be used in the above string
+        },
+    },
+
+    # Log the logs in defined files
+    'handlers': {
+        'query_log': {
+            'class': 'logging.FileHandler',
+            'filename': BASE_DIR / 'shop/logs/query.log',
+            'formatter': 'standard',
+            'level': 'DEBUG'
+        },
+        'info_log': {
+            'class': 'logging.FileHandler',
+            'filename': BASE_DIR / 'shop/logs/info.log',
+            'formatter': 'standard',
+            'level': 'INFO'
+        },
+        'error_log': {
+            'class': 'logging.FileHandler',
+            'filename': BASE_DIR / 'shop/logs/error.log',
+            'formatter': 'standard',
+            'level': 'WARNING'
+        },
+        'rollbar': {
+            'class': 'rollbar.logger.RollbarHandler',
+            'formatter': 'standard',
+            'level': 'WARNING',
+            'environment': 'development' if DEBUG else 'production',
+            'access_token': os.getenv('ROLLBAR_ACCESS_TOKEN')
+        },
+    },
+
+    # will send the logs to the handlers, filter the log levels
+    'loggers': {
+        'django': {
+            'level': os.getenv('LOG_LEVEL'),
+            'handlers': ['info_log', 'error_log', 'rollbar'],
```

Comment:
use rollbar only in production/ development not in local

Reply:
in feature we wil implement 2 branches for development and production which will resolve this issue